app/index.html

```diff
@@ -4,10 +4,32 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/reset.css">
     <link rel="stylesheet" href="styles/main.css">
     <title>Portfolio</title>
 </head>
 <body>
-    
+    <header>
+        <div class="center align-left" id="logo">
+            <span id="logo-text" class="primary-text-color">Welcome</span>
+        </div>
+        <nav class="center align-right">
+            <img id="menu" src="assets/menu_button.png" alt="">
+        </nav>
+    </header>
+    <div class="content">
+        <h2 class="center-text primary-text-color main-heading">Hey there. I&apos;m Mariane <br><span class="secondary-text-color">I&apos;m a software developer</span> </h2>
```

Comment:
- [ ]  Please make sure to use the `<h1>` header instead of a `<h2>` tag as this is one of the requirements of the project and using a more semantic tag is useful for [SEO ](https://searchengineland.com/guide/what-is-seo) reasons 🔍 .

Reply:
changed <h2> to <h1> tag.